Jenkinsfile

```diff
@@ -87,6 +93,25 @@
     }
 }
 
+def String startLogCatCollector() {
+    sh '''adb logcat -c
+    `adb logcat > "logcat.txt" &` & echo $1 > pid
+    '''
+    return readFile("pid").trim()
+}
+
+def stopLogCatCollector(String backgroundPid, boolean archiveLog) {
+    sh 'kill $backgroundPid'
```

Comment:
I hope SIGTERM will be enough in all cases. But I would of course prefer not to use SIGKILL.